README.md

```diff
@@ -8,7 +8,7 @@ Caffe, Rust and numerous research papers and brings modularity, performance and
 portability to deep learning. Leaf is lean and tries to introduce minimal
 technical debt to your stack.
 
-Leaf is a few months old, but thanks to its architecture and Rust already one of
+Leaf is a few months old, but thanks to its architecture and Rust it's already one of
```

Comment:
I think I just misread it the first time. I think it's grammatically correct, but somehow doesn't quite roll of the tongue for me. I reverted that change, however.